src/main/java/com/google/sps/data/DataUtil.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+import com.google.gson.Gson;
+
+public class DataUtil {
```

Comment:
the class name is pretty generic and parser is not used anywhere. Just rename this class videoState itself.